docs/docs.json

```diff
@@ -188,8 +187,15 @@
               "v3/how-to-guides/workflows/pass-inputs",
               "v3/how-to-guides/workflows/access-runtime-info",
               "v3/how-to-guides/workflows/run-work-concurrently",
-              "v3/how-to-guides/workflows/form-building"
-
+              "v3/how-to-guides/workflows/form-building",
+              "v3/how-to-guides/workflows/run-background-tasks"
+            ]
+          },
+          {
+            "group": "Background tasks",
```

Comment:
this group is no longer being introduced (yet), right?

Reply:
Yeah, I put the background task how-to guide under "Workflows". That doesn't seem quite right, but I think it'll work for now.